snmp/README.md

```diff
@@ -0,0 +1,51 @@
+# SNMP Monitoring Template
+
+Provided by: [bonitoo.io](.)
+
+This template provides several dashboards showing metrics provided via SNMP protocol. It provides both an example of system SNMP stats and examples from Mikrotik and Cisco devices.
+
+### Dashboard examples
+
+![Screenshot](img/snmp-dashboard.png)
+
+### Quick Install
+
+If you have your InfluxDB credentials [configured in the CLI](Vhttps://v2.docs.influxdata.com/v2.0/reference/cli/influx/config/), you can install this template with:
+
+```
+influx pkg -u https://raw.githubusercontent.com/influxdata/community-templates/master/snmp/snmp.yml
+```
+
+## Included Resources
+
+This template includes the following:
+
+    - 1 Bucket: `snmp`, 1d retention
+    - 3 Labels: `inputs.net`,`inputs.system`,`SNMP`
+    - 3 Telegraf Configurations
+    - 0 Checks: 
+    - 3 Dashboards: `SNMP System Monitoring`, `SNMP Mikrotik Monitoring`, `SNMP Network Monitoring`
+    - 2 Variables: `snmp_host`, and `snmp_ifName`
+
+## Setup Instructions
```

Comment:
Your Telegraf configs use environment variables for the InfluxDB v2 output, these variables should be listed in your Setup Instructions.